pyproject.toml

```diff
@@ -138,18 +117,15 @@ ignore = [
     "D401",  # First line of docstring should be in imperative mood
     "D407",  # Missing dashed underline after section
     "D416",  # Section name ends in colon
+    "E501",  # Line too long
 ]
 
 [tool.ruff.lint.pydocstyle]
 convention = "google"
 
 [tool.ruff.lint.per-file-ignores]
-"nautobot_dev_example/migrations/*" = [
-    "D",  # pydocstyle
-]
-"nautobot_dev_example/tests/*" = [
-    "D",  # pydocstyle
-]
+"nautobot_dev_example/migrations/*" = ["D", "F", "E", "W"]
+"nautobot_dev_example/tests/*" = ["D", "S"]
```

Comment:
I prefer to keep each item on the single line, to have better diffs.

Overall, would be nice (not in this PR) to reformat all lists in `pyproject.toml` to use this format.